### PR TITLE
Add some useful methods

### PR DESCRIPTION
**Pull request by tgallice**

### CODE REVIEW

File: lib/upfluence/endpoint/api_endpoint.rb [hunk 1]
```diff
@@ -38,6 +39,18 @@ def respond_with(resource, *args)
           halt [status, result] || 404
         end
       end
+
+      def json_params
+        begin
+          JSON.parse(request.body.read)
```
Comment (AlexisMontagne) [inline lib/upfluence/endpoint/api_endpoint.rb]: We could wrap this line in a `ActiveSupport::HashWithIndifferentAccess` class (and include `active_support` to the deps)  to have sweeter way to access to the attrs

Comment (tgallice) [reply lib/upfluence/endpoint/api_endpoint.rb]: The think is, rbutils should make `active_support` dependent all packages who using it ?

Comment (AlexisMontagne) [reply lib/upfluence/endpoint/api_endpoint.rb]: Because you used `active_model_serializer` in #2, it's already un underlied dependency

Comment (tgallice) [reply lib/upfluence/endpoint/api_endpoint.rb]: hmm good catch ! It's good to you to add deps ?

Comment (AlexisMontagne) [reply lib/upfluence/endpoint/api_endpoint.rb]: We are already using it in most (/all) of our project, i think we can safely add it to this lib

Comment (tgallice) [reply lib/upfluence/endpoint/api_endpoint.rb]: @AlexisMontagne done

File: rbutils.gemspec [hunk 2]
```diff
@@ -28,4 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_runtime_dependency 'upfluence-thrift', '~> 1.0', '>= 1.0.3'
+  spec.add_runtime_dependency 'active_model_serializers'
```
Comment (AlexisMontagne) [inline rbutils.gemspec]: You also define sinatra and active support here

File: lib/upfluence/endpoint/api_endpoint.rb [hunk 3]
```diff
@@ -38,6 +39,18 @@ def respond_with(resource, *args)
           halt [status, result] || 404
         end
       end
+
+      def json_params
+        begin
+          JSON.parse(request.body.read).with_indifferent_access
```
Comment (AlexisMontagne) [inline lib/upfluence/endpoint/api_endpoint.rb]: I think you will need to require 'active_support/core_ext/hash' before using the function

**Comment by tgallice** [comment]
@AlexisMontagne Thank ! It's a bit embarrassing to missing that :D

**Comment by AlexisMontagne** [comment]
LGTM

File: lib/upfluence/polyfill/active_model_serializers.rb [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# Add support of get_serializer_for for active_model_serializers >= 0.8.3
+module ActiveModel
+  class Serializer
+    def self.serializer_for(resource, options = {})
```
Comment (jlevesy) [inline lib/upfluence/polyfill/active_model_serializers.rb]: You can use a `class << self  end` block to define static methods

**Review by jlevesy — COMMENTED** [review]
I don't understand why we need  that `polyfill/active_mode_serializers', can you  explain ?

**Comment by tgallice** [comment]
@jlevesy I have add it at first because in `identity-server` we used `active_model_serializers:0.8.3` which not have the `serializer_for` (>0.9.5). This method is used in https://github.com/upfluence/rbutils/pull/3/files#diff-eb5389e78f05503643a4163c5d4cb7daR38 to automatically handle `ActiveModel::Serializer` class. Since, I have bump to the `v0.9.5` but I kept it just in case. But you right I think we can remove it:)

**Comment by jlevesy** [comment]
Sounds good to me

**Comment by AlexisMontagne** [comment]
@tgallice we can avoid having a polyfill and just add `gem.add_dependency 'active_model_serializers', '~> 0.9.0'` it will break the `active_model_serializers:0.8.x` deps, but i don't think we have hard depency on it

**Comment by AlexisMontagne** [comment]
LGTM

**Pull request merged**